src/Uno.UI.RuntimeTests/Tests/Uno_UI_Xaml_Core/Given_InputManager.cs

```diff
@@ -250,7 +250,7 @@ public async Task When_ProtectedCursor_PointerCaptured()
 		var button1 = (Button)stackPanel.Children[0];
 		var button2 = (Button)stackPanel.Children[1];
 
-		SetProtectedCursor(button1, InputSystemCursor.Create(InputSystemCursorShape.IBeam));
+		button1.SetProtectedCursor(InputSystemCursor.Create(InputSystemCursorShape.IBeam));
```

Comment:
I'd keep `SetProtectedCursor` as a method in RuntimeTests so we can call it when running WinUI SamplesApp.

Reply:
It doesn't really matter. The entire class is already skia-only. If we change that in the future, we can add 2-liner back.